### DISCUSSION TITLE
0.6.0 - Docco

### DISCUSSION THREAD

**Discussion by progrium** [treehousedev/treehouse#273] [Announcements]
## New Document and Tab Views
In this release we're introducing two new views: Document view, and Tab view. Views are a powerful idea in Treehouse that can either be used to create embedded mini-apps or act as building blocks for you to make your own. These new views are an example of each.

Document view gives you a more conventional note-taking experience. In Document view each child node is rendered as a paragraph in the center of the panel. Documents support simple Markdown formatting like bold, italic, strikethrough, and ordered and unordered lists. Turn a node into a document by choosing "Make Document" from the Command Palette. Unlike other views, changing to Document view is a one-way action and nodes can't be converted back to other views.

Tab view renders each child node as a tab. Their children (grandchild nodes) are shown beneath when the tab is selected. These nodes will retain their view, so you can combine Tab view with others to create a more complex larger view of your data. In general, Tab view is useful for saving vertical space across several categories. Turn nodes into tabs by selecting the parent node and choose "View as Tabs" from the Command Palette. Since Tab view is read-only, you can modify tabs by switching back to List view.

We've also renamed Live Search Nodes to Smart Nodes, which can now be named separate from their query. This also resolves an issue where previously you couldn't use them for tag searches.

[![Watch Demo](http://i3.ytimg.com/vi/-CsRlyJx2cU/hqdefault.jpg)](https://www.youtube.com/watch?v=-CsRlyJx2cU)
[Demo video](https://www.youtube.com/watch?v=-CsRlyJx2cU)

## Enhancements and Chores
* Tabs view [#126](https://github.com/treehousedev/treehouse/issues/126)
* Document view [#246](https://github.com/treehousedev/treehouse/issues/246)
* Support setting a name for a Smart Node [#235](https://github.com/treehousedev/treehouse/issues/235)
* Clean up command palette commands [#242](https://github.com/treehousedev/treehouse/issues/242)
* Rename live search/search node to Smart Node [#248](https://github.com/treehousedev/treehouse/issues/248)

## Bugfixes
* Range error when turning a tag into a search node [#236](https://github.com/treehousedev/treehouse/issues/236)
* Don't allow outdenting a top-level node [#234](https://github.com/treehousedev/treehouse/issues/234)
* Can't interact with an empty node in Quick Add (mobile) [#204](https://github.com/treehousedev/treehouse/issues/204)
* Capital letters in GitHub username prevent login [#245](https://github.com/treehousedev/treehouse/issues/245)
* Can't select a theme in Chrome [#266](https://github.com/treehousedev/treehouse/issues/266)

<hr /><em>This discussion was created from the release <a href='https://github.com/treehousedev/treehouse/releases/tag/0.6.0'>0.6.0 - Docco</a>.</em>

**Comment by ognsya** [comment]
Hello! I'm not a developer, but happy to help with feedback.

I know any feedback related to the UI and any bugs I find would probably be helpful.

However I read in your description that Treehouse is a "front end". I'm not quite sure what that means (from a user perspective)... Where does the front end... ends, and the back end begins? In other words, what type of feedback wouldn't be relevant to you?

For example, I just tried pasting a wikipedia article into Treehouse and came across a series of issues, including disappearing content. I'm guessing this is related to back end?
